subsys/bluetooth/controller/hci/hci.c

```diff
@@ -476,6 +476,29 @@ static void write_auth_payload_timeout(struct net_buf *buf,
 }
 #endif /* CONFIG_BT_CTLR_LE_PING */
 
+#if defined(CONFIG_BT_CTLR_PERIPHERAL_ISO) || \
+	defined(CONFIG_BT_CTLR_CENTRAL_ISO)
```

Comment:
Don't we need this for `CONFIG_BT_CTLR_ADV_ISO` and  `CONFIG_BT_CTLR_SYNC_ISO` as well? 

Should we consider having some aggregated ISO value to make such check easier?

Reply:
I moved to a separate section that guards by an OR of all 4 possible values